version.py

```diff
@@ -9,13 +14,26 @@
 # Module type for PyPI keywords
 # Used for:
 # - PyPI keywords
-__module_type__ = u"backend"
+__module_types__ = u"backend"
 
 # Application manifest
-__version__ = u"0.2.15"
-__author__ = u"Alignak team"
-__author_email__ = u"d.durieux@siprossii.com"
-__copyright__ = u"(c) 2015-2016 - %s" % __author__
+__version__ = u"0.3.2"
+__author__ = u"David Durieux"
+__author_email__ = u"david.durieux@alignak.net"
+__copyright__ = u"(c) 2015-2017 - %s" % __author__
 __license__ = u"GNU Affero General Public License, version 3"
-__url__ = u"https://github.com/Alignak-monitoring-contrib/alignak-module-backend"
+__git_url__ = u"https://github.com/Alignak-monitoring-contrib/alignak-checks-wmi"
```

Comment:
error, it must be `https://github.com/Alignak-monitoring-contrib/alignak-module-backend`